src/lamplib/src/tests/test_auto_tasks.py

```diff
@@ -430,5 +434,38 @@ def test_patch_tasks(self):
         )
 
 
+def test_generate_all_tasks():
```

Comment:
Could we rename this to `test_dry_run_all_tasks` or so? There's already a `test_all_tasks` further up that might be confusing.